src/elements/content-sidebar/__tests__/SidebarNavSign.test.tsx

```diff
@@ -1,80 +1,75 @@
 import * as React from 'react';
-import { render, fireEvent } from '@testing-library/react';
+import { render, screen, fireEvent } from '@testing-library/react';
+import { IntlProvider } from 'react-intl';
 import SidebarNavSign from '../SidebarNavSign';
 // @ts-ignore Module is written in Flow
 import FeatureProvider from '../../common/feature-checking/FeatureProvider';
+// @ts-ignore Module is written in Flow
+import messages from '../messages';
+// @ts-ignore Module is written in Flow
+import localize from '../../../../test/support/i18n.js';
+
+jest.unmock('react-intl');
 
 describe('elements/content-sidebar/SidebarNavSign', () => {
     const onClickRequestSignature = jest.fn();
     const onClickSignMyself = jest.fn();
+    const signLabel = localize(messages.boxSignRequestSignature.id);
+    const requestSignatureButtonText = localize(messages.boxSignRequestSignature.id);
+    const signMyselfButtonText = localize(messages.boxSignSignMyself.id);
 
     const renderComponent = (props = {}, features = {}) =>
         render(
             <FeatureProvider features={features}>
```

Comment:
dont need ot do this when you import from https://github.com/box/box-ui-elements/blob/master/src/test-utils/testing-library.tsx